### PR TITLE
added irida.db.profile to docs

### PR DESCRIPTION
**Pull request by tom114**
## Description of changes
`irida.db.profile` somehow didn't get added to the docs for this last upgrade.  Added it to the docs in the appropriate places and added to `UPGRADING.md` file.

Note this is a hotfix merge and should be merged do `master` and `development`.

## Related issue
N/A

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [x] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
~* [ ] Tests added (or description of how to test) for any new features.~
* [x] User documentation updated for UI or technical changes.

### CODE REVIEW

File: doc/administrator/web/index.md [hunk 1]
```diff
@@ -175,9 +175,16 @@ The different application profiles and their functions are the following:
 
 To launch an IRIDA application server with one (or more) of these profiles, you must enable the profile with the `spring.profiles.active` variable in your Tomcat configuration.  For example to run an IRIDA server with the `web` and `analysis` profiles active, you would set the following configuration:
 
-`spring.profiles.active=web,analysis`
+```
+spring.profiles.active=web,analysis
+```
+
+Using the multi server configuration it's also suggested you use the `irida.db.profile` setting in Tomcat:
+```
```
Comment (apetkau) [inline doc/administrator/web/index.md]: Can you add a newline here?

File: doc/administrator/web/index.md [hunk 2]
```diff
@@ -175,9 +175,16 @@ The different application profiles and their functions are the following:
 
 To launch an IRIDA application server with one (or more) of these profiles, you must enable the profile with the `spring.profiles.active` variable in your Tomcat configuration.  For example to run an IRIDA server with the `web` and `analysis` profiles active, you would set the following configuration:
 
-`spring.profiles.active=web,analysis`
+```
+spring.profiles.active=web,analysis
+```
+
+Using the multi server configuration it's also suggested you use the `irida.db.profile` setting in Tomcat:
```
Comment (apetkau) [inline doc/administrator/web/index.md]: Could you describe what `irida.db.profile` does?  E.g., will IRIDA break if it's not set?

File: doc/administrator/web/index.md (hunk 1 above)
Comment (tom114) [reply doc/administrator/web/index.md]: Done in e1663c0

File: doc/administrator/web/index.md (hunk 2 above)
Comment (tom114) [reply doc/administrator/web/index.md]: Done in e1663c0

**Comment by tom114** [comment]
Also pulled the version number back in cfc624a.  Since it's just a docs fix no need for a release.  Just hotfix and we'll release the docs.

**Pull request merged**